scripts.py

```diff
@@ -1880,3 +1883,77 @@ def do_run(self, output=sys.stdout, cmd_args=None, exporter=None):
 
         exporter = exporter or LocalAnalyticsExporter()
         output.write(exporter.export(self._db, start, end))
+
+class GenerateShortTokenScript(LibraryInputScript):
+    """
+    Generate long lived client short tokens that can be used for testing the service. This
```

Comment:
Minor: This script can generate tokens of widely varying lengths (1 minute and up). I'd suggest the following wording for the first sentence: "Generate a short client token of the specified duration that can be used for testing that involves the Adobe Vendor ID API implementation."